netty/src/main/java/io/grpc/netty/RequestMessagesCommand.java

```diff
@@ -1,52 +0,0 @@
-/*
- * Copyright 2014, Google Inc. All rights reserved.
- *
- * Redistribution and use in source and binary forms, with or without
- * modification, are permitted provided that the following conditions are
- * met:
- *
- *    * Redistributions of source code must retain the above copyright
- * notice, this list of conditions and the following disclaimer.
- *    * Redistributions in binary form must reproduce the above
- * copyright notice, this list of conditions and the following disclaimer
- * in the documentation and/or other materials provided with the
- * distribution.
- *
- *    * Neither the name of Google Inc. nor the names of its
- * contributors may be used to endorse or promote products derived from
- * this software without specific prior written permission.
- *
- * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
- * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
- * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
- * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
- * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
- * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
- * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
- * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
- * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
- * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
- * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- */
-
-package io.grpc.netty;
-
-import io.grpc.internal.AbstractStream2;
-
-/**
- * Command which requests messages from the deframer.
- */
-class RequestMessagesCommand extends WriteQueue.AbstractQueuedCommand {
-
-  private final int numMessages;
-  private final AbstractStream2.TransportState state;
-
-  public RequestMessagesCommand(AbstractStream2.TransportState state, int numMessages) {
-    this.state = state;
-    this.numMessages = numMessages;
-  }
-
-  void requestMessages() {
-    state.requestMessagesFromDeframer(numMessages);
```

Comment:
I might be missing something, but `requestMessagesFromDeframer` is defined in class `io.grpc.internal.AbstractStream2.TransportState` and called in class `io.grpc.netty.NettyClientStream.Sink` ... I don't see how anything but `public` would work?